client/src/featureform/online.py

```diff
@@ -0,0 +1,174 @@
+from abc import ABC, abstractmethod
+from typing import List, Any, Union
+
+import numpy as np
+from .enums import ScalarType
+from .resources import PineconeConfig
+import pinecone
+import uuid
+
```

Comment:
I'm thinking the package structure for this should be

-online_store
--online.py (interfaces to implement)
--pinecone.py
--weaviate.py
--redis.py
--other_online_store.py

main point is that as we add online stores i don't think they should all be in 1 file.